getter.go

```diff
@@ -144,6 +144,7 @@ func (g *getter) queueFile(url *url.URL) (http.Header, error) {
 	// resp could be nil, depending on the error.
 	if err != nil {
 		g.chunkWg.Done()
+		defer resp.Body.Close()
```

Comment:
The chunk gets sent over ```g.getCh```, which in the ```worker()``` goroutines, gets passed to ```g.retryGetChunk()```, which goes to ```g.getChunk```, which [has this line](https://github.com/hasnickl/s3gof3r/blob/master/getter.go#L275):
```defer checkClose(resp.Body, &err)```

source:
```
func checkClose(c io.Closer, err *error) {
	cerr := c.Close()
	if *err == nil {
		*err = cerr
	}
}
```